_posts/2017-5-31-release-1.3.0.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title:  "Apache Flink 1.3.0 Release Announcement"
+date:   2017-05-31 19:00:00
+author: "Robert Metzger"
+author-twitter: "rmetzger_"
+categories: news
+---
+
+
+The Apache Flink community is pleased to announce the 1.3.0 release. Over the past 4 months, the Flink community has been working hard to resolve more than 680 issues. See the [complete changelog]() for more detail.
+
+This is the fourth major release in the 1.x.y series. It is API compatible with the other 1.x.y releases for APIs annotated with the @Public annotation.
+
+We encourage everyone to download the release and check out the [documentation](https://ci.apache.org/projects/flink/flink-docs-release-1.3/). Feedback through the [Flink mailing lists](http://flink.apache.org/community.html#mailing-lists) is, as always, gladly encouraged!
+
+You can find the binaries on the updated [Downloads page](http://flink.apache.org/downloads.html). Some highlights of the release are listed below.
+
+
+# Large State Handling / Recovery
+
+* **Incremental Checkpointing for RocksDB**: It is now possible to checkpoint only the difference from the previous successful checkpoint, rather than checkpointing the entire application state ([FLINK-5053](https://issues.apache.org/jira/browse/FLINK-5053))
+
+* **Asynchronous snapshots for heap-based state backends**: The filesystem and memory statebackends now also support asynchronous snapshots using a copy-on-write HashMap implementation. Asynchronous snapshotting makes Flink more resilient to slow storage systems and expensive serialization. The time an operator blocks on a snapshot is reduced to a minimum. ([FLINK-6048](https://issues.apache.org/jira/browse/FLINK-6048), [FLINK-5715](https://issues.apache.org/jira/browse/FLINK-5715))
+
+* **Allow upgrades to state serializers:** Users can now upgrade serializers, while keeping their application state. One use case of this is upgrading custom serializers used for managed operator state / keyed state. Also, registration order for POJO types / Kryo types is now no longer fixed ([FLINK-6178](https://issues.apache.org/jira/browse/FLINK-6178))
```

Comment:
quoting from the link you gave: "It is, however, common to include a space on each side of the slash when it connects items which themselves have spaces", so here, it is better with spaces IMHO